docs/sphinx/releases/whats_new_1_5_0.rst

```diff
@@ -8,17 +9,542 @@
+  easier for the user. ``reduce``, ``copy``, ``all_of``/``any_of``/``none_of``,
+  ``fill``, ``count``, ``destroy``, ``move`` are using those CPOs.  We also have
+  adapted their corresponding ``hpx::ranges`` versions in this release,
+  deprecating the ``hpx::parallel`` ones.
+* In the perspective of continuing our work on making the hpx algorithms support
+  the ranges, we have adapted ``hpx::traits::is_range`` and others to use
+  ``hpx::traits::is_sentinel_for`` concept. We have adapted ``hpx::copy``
+  and ``hpx::stable_sort`` algorithms.
+* We have adapted ``co_await`` to C++20, it now supports ``shared_future`` and
+  have added allocator support for futures returned by ``co_return``. It is no
+  longer in the ``experimental`` namespace.
+* We added serialization support for ``std::variant`` and ``std::tuple``.
+* We continued with the modularization, it is now easier to enable/disable the
+  distributed runtime option. An important amount of headers have been
+  deprecated. We adapted the namespaces and headers we could to be closer to the
+  standard ones (:ref:`api <_public_api>`).
+* We have set the HPX C++ standard automatically to C++14 and if it needs to be
+  set explicitly, it should be through the ``CMAKE_CXX_STANDARD``. The
+  ``HPX_WITH_CXX*`` variables are now deprecated.
+* It is now possible to have a basic CUDA support including a helper function to
+  get a future from a CUDA stream and target handling. They are available under
+  the ``hpx::cuda::experimental`` namespace and they can be enabled with the
+  ``HPX_WITH_ASYNC_CUDA=ON`` cmake option.
+* We added a new ``hpx::mpi::experimental`` namespace for getting futures from
+  an asynchronous MPI call and a new minimal MPI executor (add the name).
+* We have added a ``hpx::init_params`` struct to pass parameters for HPX
+  initialization e.g. the resource partitioner callback to initialize thread
+  pools (:ref:`using the resource partitioner from hpx 1.5.0 onwards
+  <_using_rp_from_1_5_0>`).
+* A polymorphic executor has been implemented to reduce compile times as a
+  function accepting executor can potentially be instanciated only once instead
+  of multiple times with different executors. It accepts the function signature
+  as a template argument. It needs to be constructed from any other executor.
+  Please note, that the function signatures that can be scheduled using
+  ``then_execute``, ``bulk_sync_execute``, ``bulk_async_execute`` and
+  ``bulk_then_execute`` are slightly different.
+* The underlying executor of ``block_executor`` has been updated to a newer one.
+* We have added a parameter to ``auto_chunk_size`` to control the amount of
+  iterations to measure.
+* We have added performance counters type information (aggregating,
+  monotonically increasing, average count, average timer, etc.).
+* The tasks are now resumed on the worker thread they were suspended to avoid
+  cache misses from moving from one thread to the other.
```

Comment:
Is this a general rule? That would essentially disable work-stealing. Or did you mean to say something different?

Reply:
Yeah, maybe this wording is slightly too strong. Tasks are re-*scheduled* on the same worker thread where they were suspended. It doesn't stop the task from being stolen though if the worker thread doesn't get to executing it before another worker thread gets that far.

Reply:
Is this ok?
```
+* The tasks are now re-scheduled on the same worker thread they were suspended
+  on to avoid cache misses from moving from one thread to the other. This
+  behavior doesn't prevent the task from being stolen."
```

Reply:
Sounds good to me. Thanks!